lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1792,6 +1792,8 @@ namespace Js
             Wasm::BaseWasmReader *reader = nullptr;
             Wasm::WasmBytecodeGenerator *bytecodeGen = nullptr;
 
+            bool traceReader = PHASE_TRACE1(WasmReaderPhase);
```

Comment:
Is it possible to move this into WasmBinaryReader and dispense with m_trace altogether?